mobile/src/app/app.component.ts

```diff
@@ -28,7 +28,7 @@ export class AppComponent {
       .subscribe({
         next: data => {
           this.statusBar.styleDefault();
-          this.router.navigate([data[1] ? 'home/tabs/tab1' : 'login']);
+          this.router.navigate(['home/tabs/map']);
```

Comment:
@calca ma così non si fa mai il login e il settaggio del profilo iniziale?

Reply:
bisogna trovare un punto di ingaggio, nel terzo tab ?

Reply:
@calca qualche indizio in più ? :D
la mia idea era:
Apro app  => prima login ? Login => setta profilo : map
Apro app => già effettuata login => map

Reply:
No no

- apro app, vedo mappa con gli helper vicino
- clicco sull'avatar (o facciamo uno slider do invito) se non sono loggato, faccio login

Può andare ?

@Al ci fai 3 slide per spiegare le funzionalità dell'app al primo avvio ?

Reply:
@calca facciamo vedere i dati degli utenti a tutti anche a chi non è registrato

Reply:
Sì, chi ha bisogno di aiuto non si deve per forza registrare, no ?